_docs_integrate/61-data-model.md

```diff
@@ -32,17 +32,17 @@ Tokens can be used to save arbitrary structured data on the Backbone, which is e
+| Name               | Type      | Description                                                                                                                                                                                 | Remarks                                       |
+| ------------------ | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------- |
+| id                 | `string`  | {% include descr_id class="Token" prefix="TOK" %}                                                                                                                                           |
+| createdBy          | `string`  | {% include descr_createdBy class="Token" %}                                                                                                                                                 |                                               |
+| createdByDevice    | `string`  | {% include descr_createdByDevice class="Token" %}                                                                                                                                           |                                               |
+| content            | `unknown` | The content of the Token. You can add whatever you want here.                                                                                                                               | will be encrypted before sent to the Backbone |
+| createdAt          | `string`  | {% include descr_createdAt class="Token" %}                                                                                                                                                 |                                               |
+| expiresAt          | `string`  | {% include descr_expiresAt class="Token" %}                                                                                                                                                 |                                               |
+| secretKey          | `string`  | {% include descr_secretKey class="Token" %}                                                                                                                                                 | saved only locally                            |
+| truncatedReference | `string`  | {% include descr_truncatedReference class="Token" %}                                                                                                                                        | saved only locally                            |
+| isEphemeral        | `boolean` | If set to `true` the token will not be cached in the database and only displayed once. You will not be able to fetch this Token unless you remember the id and secretKey of the Token. |
```

Comment:
da steht jetzt 2x Token hintereinander, mein Fehler.

`unless you remember the id and secretKey of the Token` -> `unless you remember its id and secretKey`